### PR TITLE
Added alternate controller w/ bindings

### PR DESCRIPTION
**Pull request by ethanglocke**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -77,6 +77,12 @@ public static class ControllerConstants{
     public static int firstAuto = 12;
     public static int secondAuto = 10;
     public static int thirdAuto = 8;
+
+    public static int OpenClaw = 2;
+    public static int CloseClaw = 3;
+    public static int RetractArms = 4;
+    public static int ExtendIntake = 5;
+    public static int RetractIntake = 6;
```
Comment (josephhadad05) [inline src/main/java/frc/robot/Constants.java:85]: When naming variables in java you typically use camelCase as opposed to PascalCase. You can read more about that here: https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -44,7 +47,7 @@ public class RobotContainer {
   private final Claw claw = new Claw();
 
   private final DriverController mainController = new DriverController(Constants.DeviceIds.driverController);
-
+  private final DriverController alternaController = new DriverController(Constants.DeviceIds.driverController);
```
Comment (josephhadad05) [inline src/main/java/frc/robot/RobotContainer.java:50]: You used the controller id for the `mainController`, you need to create a new one for the `alernateController`.

Also, you misspelled alternate.

**Review by josephhadad05 — CHANGES_REQUESTED** [review]

**Review by josephhadad05 — APPROVED** [review]
Looks good.

**Review by josephhadad05 — APPROVED** [review]

**Review by paulelliott — APPROVED** [review]
Looks great! Well done!

**Pull request merged**